app/server/utils/ussd/kenya_ussd_state_machine.py

```diff
@@ -241,23 +243,23 @@ def process_exchange_token_request(self, user_input):
         amount = float(self.session.get_data('exchange_amount'))
         ussd_tasker.exchange_token(self.user, agent, amount)
 
+    @staticmethod
+    def make_usage_mapping(usage: TransferUsage):
+        return {'id': usage.id, 'translations': usage.translations, 'name': usage.name}
+
     def store_transfer_usage(self, user_input):
-        items_per_menu = 8
-        usages = self.user.get_most_relevant_transfer_usage()
-        transfer_usage_id_order = []
-        for usage in usages:
-            transfer_usage_id_order.append(usage.id)
-        if self.session.session_data is None:
-            self.session.session_data = {'transfer_usage_mapping': transfer_usage_id_order}
-        elif type(self.session.session_data) is dict:
-            self.session.session_data['transfer_usage_mapping'] = transfer_usage_id_order
-        self.session.set_data('usage_menu', 1)
```

Comment:
@RubenGeo off by 1 error if you navigated from `send_transfer_reason` to `send_transfer_reason_other` hehe